### DISCUSSION TITLE
Redux migration - state management doubt

### DISCUSSION THREAD

**Discussion by marcos-smartflow** [TanStack/query#7397] [Q&A]
is there a way to use a simple function to set selected accounts using React Query? this is what I have using redux, I need something similar and be able to read selectedAccounts from anywhere in the code

 const setSelectedAccounts = React.useCallback(
    (selectedAccounts: Account[]) =>
      dispatch({ type: ActionTypes.SELECT_ACCOUNTS, payload: selectedAccounts }),
    [dispatch],
  )

case ActionTypes.SELECT_ACCOUNTS:
      return { ...state, selectedAccounts: action.payload, loading: false, error: false }

**Comment by TkDodo** [comment]
selected accounts are client state - they have nothing to do with a sever resource, so you shouldn't be managing this with react query.
